apps/studio/next.config.mjs

```diff
@@ -74,7 +74,11 @@ const ContentSecurityPolicy = `
     https://*.wogaa.sg
     https://placehold.co
     https://cdn.growthbook.io
-    ${env.NODE_ENV === "production" ? "https://isomer-user-content.by.gov.sg" : "https://*.by.gov.sg"}
+    ${
```

Comment:
Let's add verification steps section in PR's description

Reply:
oh i was thinking this is a follow-up/hotfix of https://github.com/opengovsg/isomer/pull/902 so don't need, but will do!